### PR TITLE
fix(head): flip encoder on C2

### PR DESCRIPTION
**Pull request by sfoster1**
Encoder goes the wrong direction now because the gears flipped and the encoder is on the motor axle.

## Testing
- [x] See if the encoder goes the right direction on C2
- [x] See if the encoder goes the right direction on B and previous

### CODE REVIEW

**Review by fsinapi — APPROVED** [review]
LGTM if it works

File: head/firmware/motor_hardware_common.c [hunk 1]
```diff
@@ -231,14 +231,21 @@ void MX_GPIO_Init(void) {
     initialize_rev_specific_pins();
 }
 
+#if PCBA_PRIMARY_REVISION == 'b' || PCBA_PRIMARY_REVISION == 'a'
+    static const int encoder_direction = TIM_ICPOLARITY_RISING;
```
Comment (Laura-Danielle) [inline head/firmware/motor_hardware_common.c:235]: The direction is flipped for both motors?

**Review by Laura-Danielle — APPROVED** [review]
had a q, but lgtm in general

**Review by ahiuchingau — APPROVED** [review]
🧭

File: head/firmware/motor_hardware_common.c (hunk 1 above)
Comment (sfoster1) [reply head/firmware/motor_hardware_common.c]: yeah, both motors need the change right?

Comment (sfoster1) [reply head/firmware/motor_hardware_common.c]: since they both changed the number of gears

**Pull request merged**